### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by HerbivorousBird** [community/community#122483] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I am currently experiencing the missing token problem for quite some time now. I have already scourged through discussions looking for hints to make it without success. I have refreshed my vscode, uninstalled and installed it again, cleared it's cache, and tried signing out and signing in back again multiple times but nothing is working. I would appreciate it if the team could fix this issue.

2024-04-05 14:14:13.043 [info] [auth] Invalid copilot token: missing token: 403 
2024-04-05 14:14:13.092 [error] [auth] Extension activation failed: "Contact Support. You are currently logged in as HerbivorousBird."

**Comment by davecheney** [comment, marked as answer]
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
